src/seedu/addressbook/data/person/Address.java

```diff
@@ -11,8 +11,11 @@
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
-    public final String value;
+    private String block;
+    private String street;
+    private String unit;
+    private String postal_code;
+    public String value;
```

Comment:
It's a good practice to limit all class members permission whenever possible.
Consider setting permission to private